source/process.adoc

```diff
@@ -264,14 +266,14 @@ The notions of Major, Minor, and Service Release bear a close resemblance to the
 [[efsp-releases-major]]
 ==== Major and Minor Releases
 
-A Major Release includes significant new features and/or breaking changes. A Minor Release includes new features over a Major Release. For both Major and Minor Releases the Specification Team must engage in at least one successful <<efsp-reviews-progress, Progress Review>> and a successful <<efsp-reviews-release, Release Review>>.
+A Major Release includes significant new features and/or breaking changes. A Minor Release includes new features over a Major Release. 
 
 A Specification Project's first Release must be a Major or Minor Release.
 
 [[efsp-milestones]]
 ==== Milestone Builds
 
-Leading up to a Release, a Specification Team must produce at least one Milestone Build. Milestones and Release Candidates are "almost Releases" intended for adoption and testing by early-adopters. No formal Reviews are required for Milestone Builds. 
+Leading up to a Release, a Specification Team should produce at least one Milestone Build. Milestones and Release Candidates are "almost Releases" intended for adoption and testing by early-adopters. No formal Reviews are required for Milestone Builds. 
```

Comment:
Are we talking about a Milestone Build of the specification document (and javadocs)?
How are those intended for use by early adopters?

Or was this intended to refer to Milestone Builds of the corresponding implementation(s)?

Reply:
The former. Early adopters, including those who are engaged in the creation of the initial set of Compatible Implementations, will use the Milestone Builds as the basis of their work. Naturally, no compatibility claims may be made based on a Milestone Build.

Reply:
Perhaps insert a few words...

... are "almost Releases" that, in combination with an implementation, are intended ...

Reply:
The teams building Compatible Implementations are the early adopters. 

I suppose that organizations building servlets and things might also be adopters, but that's not what I intend.

Reply:
Ya, I think of the people *using* the non-final implementations as the early adopters.
The people building the implementations are just the implementors.  I don't classify
them as "early" or "late".

I guess it's all relative, but the important adopters are the people using the implementations.
If the people building the implementations are adopters of the spec, then what do we call
the people using the implementations?

Reply:
Based on Ed's comments, I'm going to revert this change (i.e. "should" will be turned back to "must").

Coming up with the right words here is hard. I think that "implementors" is probably the best. How do we express that?

"... intended to guide and inform implementors, validate build and deployment practices, and solicit feedback." ?

Reply:
I've opened #17. Defer for now.